index.bs

```diff
@@ -2088,22 +2142,28 @@ Methods</h4>
 			3. In case of failure, queue a task on the <a>control thread</a> to execute the following,
 				and abort these steps:
 
-				1. Reject all promises from <a>pendingResumePromises</a> in order, then clear <a>pendingResumePromises</a>.
+				1. Reject all promises from {{AudioContext/[[pending resume promises]]}}
```

Comment:
An `OfflineAudioContext` isn't an `AudioContext`, so shouldn't we define `[[pending resume promises]]` slot of an offline context?  And update links accordingly.

Also, should the promise be appended to `[[pending resume promises]]` in earlier steps?  Otherwise this is empty.

Reply:
This is a mistake, I'm removing it. There is no `[[pending resume promises]]` on `OfflineAudioContext` as you note.